lib/flake.nix

```
@@ -0,0 +1,5 @@
+{
+  description = "Nixpkgs lib, a zero-dependency extension to the builtin library.";
```

Let's just use the wording in `./lib/default.nix`.
```suggestion
  description = "Library of low-level helper functions for nix expressions.";
```